.ci/jobs/nightly-build.yml

```diff
@@ -12,5 +12,5 @@
             branches:
               - master
             credentials-id: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba'
-      script-path: build/ci/nightly/Jenkinsfile
+      script-path: build/ci/release/Jenkinsfile
```

Comment:
I'm wondering if we should not add a comment here (or somewhere else) to explain that this job is using the "release" Jenkinsfile on purpose and that the nightly and the release jobs should be as much as possible kept "in sync".

Reply:
+1 to add a note.

It may also be an indication that the naming is not good. It could be `build/ci/build/Jenkinsfile` used by `nightly-build.yml` and `release-build.yml`.

Reply:
I renamed and added a note.